tests/query/injections/html/test-html-injections.html

```diff
@@ -5,38 +5,20 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="css/style.css" rel="stylesheet">
-    <style>
-    footer{
-/*    ^ css
-*/
-    }
-    </style>
-    <style title="Test Style without type attribute">
-    footer{
-/*    ^ css
-*/
-    }
-    </style>
-    <style type="text/css" title="test style with defined type attribute">
-    footer{
-/*    ^ css
-*/
-    }
-    </style>
+    <style> footer{ } </style>
+    <!--      ^ css  -->
+    <style title="Test Style without type attribute"> footer{ } </style>
```

Comment:
Due to misunderstanding, I thought before that `highlight-assertions` can use comments from injected parsers too
It was not capturing things inside the script and style tags because i use css/js-style comments
Fixed to make sure they are captured now, and they all still passed :D